lib/screens/login/login_screen.dart

```diff
@@ -50,18 +47,11 @@ class _LoginScreenState extends ConsumerState<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
-    final isLoggedIn = ref.watch(isLoggedInProvider).value ?? false;
-    if (isLoggedIn) {
-      showToast('Logged In. Will navigate to Home Screen');
-      // TODO _navigateToHome()
-      return Container();
-    }
-
     ref.listen<BaseViewModelState>(loginViewModelProvider, (_, state) {
       state.maybeWhen(
         success: () {
           showToast('Login Success');
```

Comment:
[Optional] Shall we remove this toast now? 😄

Reply:
@ryan-conway Let me remove it at part2. 😄

Reply:
@ryan-conway Removed at 838f9a51 🙏🏻